### DISCUSSION TITLE
Add filterings on what the search is outputing

### DISCUSSION THREAD

**Discussion by antoineKorbit** [dailydotdev/daily#1431] [Feature Requests]
The new search is great, with the ability to Ask the IA to get some answers. 
But unfortunatly, I see a lot of videos when searching for something in there. But I would prefer to have the ability to filter the types and get articles instead.

**Comment by idoshamun** [comment]
Hi @antoineKorbit,

Thanks for the suggestion 🙏

**Comment by sabdailydev** [comment]
@antoineKorbit  We have added filters to search results. Check it out and let us know what you think. 
<img width="571" height="351" alt="Screenshot 2025-07-14 at 10 10 03" src="https://github.com/user-attachments/assets/6f67f0aa-86b0-4e85-be84-c23bdafb3650" />
<img width="572" height="381" alt="Screenshot 2025-07-14 at 10 09 55" src="https://github.com/user-attachments/assets/66cdd390-205c-4c8e-86ee-438d89e1de4a" />
